ui/imports/shared/views/ExistingContacts.qml

```diff
@@ -83,21 +83,21 @@ Item {
 
         delegate: StatusMemberListItem {
             width: contactListView.availableWidth
-
+            readonly property bool ensVerified: Utils.isEnsVerified(model.pubKey)
```

Comment:
It would be even better if `ensVerified` was part of the model. It should be easy, since `createItemFromPublicKey` uses `getContactDetails` to populate the Item.
Also, if you do that, can you clean it up a little, we do both of those:
```nim
let contact =  self.controller.getContact(publicKey)
let contactDetails = self.controller.getContactDetails(contact.id)
```
which is useless, they basically return the same thing (only `contactDetails`  is useful, since`contactDetails.details` is the ContactDto)

Reply:
Yup, will do